lua/core/info.lua

```diff
@@ -4,15 +4,20 @@ local null_ls_handler = require "lsp.null-ls"
 local indent = "  "
 
 M.banner = {
+  indent .. "Press q to close",
```

Comment:
My only real complaint is this. I think it looks off. I tried giving it a big offset but I'm not sure it's that much better anyway
```lua
  local function get_adaptive_dismiss_hint(window_width)
    local msg = " "
    local hint = "Press q to dismiss"
    local padding = 4
    for i = 1, (window_width - (string.len(hint) + padding)) do
      msg = msg .. " "
    end
    return { "", msg .. hint, "" }
  end

  vim.list_extend(buf_lines, get_adaptive_dismiss_hint(math.floor(vim.o.columns * 0.7)))
```

Maybe we should see how Telescope embeds it in the frame.

Reply:
Removed "Press q to close"